utils/devekf/devekf.c

```diff
@@ -25,6 +26,24 @@
 
 enum printMode { prntVersor, prntAtt, printPos, silent };
 
+
+int devekf_run;
+
+
+static void *keyboardHandler(void *args)
+{
+	char input;
```

Comment:
```suggestion
	int input;
```
`getchar()` is type of `int`